### PR TITLE
[mgt-teams-tab]: Teams Tab implementing microsoft graph toolkit component

### PR DESCRIPTION
**Pull request by Trinetra-MSFT**
### Description

This sample app demonstrate the addition of microsoft graph toolkit component to an teams tab. You can integrate mgt-component in your tab and have a rich experience in Tabs. Once user sign in in Tab they can see other graph-toolkit component

**Language Used: React**

### Proposed changes

- Create a teams tab
- Add following microsoft-graph toolkit component in Tab

1. Login 
2. People-Picker
3. Agenda 
4. Tasks 

### Testing
 Covering all mgt-component scenario over here.
**mgt-login and mgt-agenda**

image

**mgt-people-picker**

image

**mgt-tasks**

image

### CODE REVIEW

File: samples/microsoft-graph-toolkit/mgt-teams-tab/.gitignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (Wajeed-msft) [inline samples/microsoft-graph-toolkit/mgt-teams-tab/.gitignore:1]: Could you please rename this to tab-graph-toolkit?

File: samples/microsoft-graph-toolkit/mgt-teams-tab/.gitignore [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+# dependencies
```
Comment (Wajeed-msft) [inline samples/microsoft-graph-toolkit/mgt-teams-tab/.gitignore:3]: Could you please add link to this from Root readme file?

File: samples/microsoft-graph-toolkit/mgt-teams-tab/.gitignore (hunk 1 above)
Comment (Wajeed-msft) [reply samples/microsoft-graph-toolkit/mgt-teams-tab/.gitignore:1]: Also, please correct the folder staructure as well. Final path should look samples/tab-graph-toolkit/node

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

File: samples/microsoft-graph-toolkit/mgt-teams-tab/.gitignore (hunk 1 above)
Comment (Wajeed-msft) [reply samples/microsoft-graph-toolkit/mgt-teams-tab/.gitignore]: @Trinetra-MSFT - I see that folder staructure is not changed to **/samples/tab-graph-toolkit/nodejs/mgt-teams-tab** , could you please remove **mgt-teams-tab** and move those files directly to root of **nodejs** folder?

File: samples/microsoft-graph-toolkit/mgt-teams-tab/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+- Once you access the Tab within your app you will be able to see following microsoft-graph-toolkit component. 
+-<mgt-login>, <mgt-agenda>, <mgt-people-picker>, <mgt-tasks>
+
+## Deploy to Teams
+Start debugging the project by hitting the `F5` key or click the debug icon in Visual Studio Code and click the `Start Debugging` green arrow button.
+
+### NOTE: First time debug step
+On the first time running and debugging your app you need allow the localhost certificate.  After starting debugging when Chrome is launched and you have installed your app it will fail to load.
+
+- Open a new tab `in the same browser window that was opened`
```
Comment (Wajeed-msft) [inline samples/microsoft-graph-toolkit/mgt-teams-tab/README.md]: @Trinetra-MSFT - could you please include steps for ngrok tunnuling + manifest changes?

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Review by asnow003 — APPROVED** [review]

**Pull request merged**